README.md

```diff
@@ -127,6 +127,10 @@ jnind4jcpu.dll: Can't find dependent libraries
 ```
 <b>Solution</b>: <br /> Install latest CUDA (version 7.6 works)
 
+<b>Problem</b>: <br /> Bad internet connection and unable to perform task smoothly.
```

Comment:
@azhar337 I don't consider internet connection a "problem", can you move this suggestion to the "usage" section above maybe?